### DISCUSSION TITLE
[docs] when to use and not use datx

### DISCUSSION THREAD

**Discussion by akaguny** [infinum/datx#1151] [Q&A] [answered]
HI, thnks for amazing work!

Pls. explain what cases this library solve.

I have tried to use it for managing very simple stores in typescript) but without decorators.
I really like commit, revert helpers. But maybe some more information about library will help me to use it in more effective way)

**Comment by DarkoKukovec** [comment, marked as answer]
Hi, the primary idea behind the library was to normalize the data (and also help with serialization for json:api). The goal is to have everything in the models that are linked together into a graph of sorts.
Of corse, since data management is related to state management, datx also has a mobx integration (it started as basic utils on top of mobx), but also has some "vanilla" and rxjs helpers - e.g. on Angular projects, we're using it with rxjs and without mobx.
